package.json

```diff
@@ -1,4051 +1,4042 @@
-            "comment.documentation.processingInstruction.cs"
-          ],
-          "xmlDocCommentText": [
-            "comment.documentation.cs"
-          ],
-          "regexComment": [
-            "string.regexp.comment.cs"
-          ],
-          "regexCharacterClass": [
-            "constant.character.character-class.regexp.cs"
-          ],
-          "regexAnchor": [
-            "keyword.control.anchor.regexp.cs"
-          ],
-          "regexQuantifier": [
-            "keyword.operator.quantifier.regexp.cs"
-          ],
-          "regexGrouping": [
-            "punctuation.definition.group.regexp.cs"
-          ],
-          "regexAlternation": [
-            "keyword.operator.or.regexp.cs"
-          ],
-          "regexSelfEscapedCharacter": [
-            "string.regexp.self-escaped-character.cs"
-          ],
-          "regexOtherEscape": [
-            "string.regexp.other-escape.cs"
-          ]
-        }
-      }
-    ],
-    "languages": [
-      {
-        "id": "aspnetcorerazor",
-        "extensions": [
-          ".cshtml",
-          ".razor"
-        ],
-        "mimetypes": [
-          "text/x-cshtml"
-        ],
-        "configuration": "./src/razor/language-configuration.json",
-        "aliases": [
-          "ASP.NET Razor"
-        ]
-      }
-    ],
-    "grammars": [
-      {
-        "language": "aspnetcorerazor",
-        "scopeName": "text.aspnetcorerazor",
-        "path": "./src/razor/syntaxes/aspnetcorerazor.tmLanguage.json"
-      }
-    ],
-    "menus": {
-      "commandPalette": [
-        {
-          "command": "dotnet.test.runTestsInContext",
-          "when": "editorLangId == csharp"
-        },
-        {
-          "command": "dotnet.test.debugTestsInContext",
-          "when": "editorLangId == csharp"
-        }
-      ],
-      "editor/title": [
-        {
-          "command": "extension.showRazorCSharpWindow",
-          "when": "resourceLangId == aspnetcorerazor"
-        },
-        {
-          "command": "extension.showRazorHtmlWindow",
-          "when": "resourceLangId == aspnetcorerazor"
-        },
-        {
-          "command": "razor.reportIssue",
-          "when": "resourceLangId == aspnetcorerazor"
-        }
-      ],
-      "editor/context": [
-        {
-          "command": "dotnet.test.runTestsInContext",
-          "when": "editorLangId == csharp",
-          "group": "2_dotnet@1"
-        },
-        {
-          "command": "dotnet.test.debugTestsInContext",
-          "when": "editorLangId == csharp",
-          "group": "2_dotnet@2"
-        }
-      ]
-    },
-    "viewsWelcome": [
-      {
-        "view": "debug",
-        "contents": "[Generate C# Assets for Build and Debug](command:dotnet.generateAssets)\n\nTo learn more about launch.json, see [Configuring launch.json for C# debugging](https://aka.ms/VSCode-CS-LaunchJson).",
-        "when": "debugStartLanguage == csharp"
-      }
-    ]
-  }
+	"name": "csharp",
```

Comment:
Why is there an 8000 line diff in this file?

Reply:
oh no. indentation went from 2 to 4. sigh

Reply:
Fixed